tests/executors/test_tcp_executor.py

```
@@ -17,42 +16,42 @@
         s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
-        return s.getsockname()[1]
+        return int(s.getsockname()[1])
 
 
-test_port = _find_free_port()
-
-
-PORT = 7986
-HTTP_SERVER = f"{HTTP_SERVER_CMD} {PORT}"
-NC_COMMAND = f'bash -c "sleep 2 && nc -lk {test_port}"'
+def nc_command(port: int, sleep_seconds: int = 2) -> str:
```

Ruff (0.14.8)

Unused function argument: `sleep_seconds`

(ARG001)